cli/commands/hcl/format/format.go

```diff
@@ -26,7 +28,13 @@ import (
 	"github.com/gruntwork-io/terragrunt/util"
 )
 
-func Run(opts *options.TerragruntOptions) error {
+var excludePathes = []string{
```

Comment:
`excludePaths`

Reply:
Let's resolve this later to get this merged.